src/dtw.jl

```diff
@@ -2,7 +2,26 @@
 "
 Probable input: Vector, Vector, distance
 "
-function dtw(fakevar::Bool)
+function dtw_test(fakevar::Bool)
     print("Hello World!")
     return fakevar
 end
+
+"
+runs the dtw algorthm on the 2 data sets then gets which values from that data
+the dtw alignements points to
+"
```

Comment:
don't understand what is this function supposed to do. What I think it does now:
1. take arbitrary data1, data2 (which can literally be anything as types are not defined)
2. run dtw from DynamicAxisWarping package
3. Iterate over the results and push them into another vector? why is it needed at all? It is fine to keep the same interface/output as DynamicAxisWarping package 

What I think this function should be doing:
1. Named nanopore_dtw(x::Vector{Float32}, y::Vector{Float32})
2. Does the fast5 read segmentation on both x, and y 
3. runs dtw on x, y with Bhattacharyya distance from Distances package
4. Returns dtw output - same as in dtw function

You can implement some simple read segmentation with change point analysis packages e.g. https://github.com/STOR-i/Changepoints.jl
Sensitivity should be a parameter, I guess you can set it to some average standard deviation from our model kmer values.

Detect change points -> this gives the set of points for each change point -> for each set of points you can calculate mean + standard deviation -> use these to calculate distance between two normal distributions https://en.wikipedia.org/wiki/Bhattacharyya_distance you can use Bhattacharyya distance
from Distances package

Reply:
Also, add a test for that function on some small random data, this will allow us to detect changes when they happen in our dependencies.

Reply:
And there is no value by having plot added to the library, just remove that file